lib/matplotlib/ticker.py

```diff
@@ -634,8 +605,9 @@ def get_offset(self):
             if self._usetex or self._useMathText:
                 sciNotStr = self.format_data(10 ** self.orderOfMagnitude)
             else:
-                # sciNotStr = '%g' % 10 **self.orderOfMagnitude
-                sciNotStr = ('\u2A2F ' + self.format_data(
+                #  According to Wikipedia this is the accepted unicode
+                # character for the multiplication sign
+                sciNotStr = ('\u00D7' + self.format_data(
```

Comment:
I am concerned about how many fonts do not support this code point (so it would get rendered as a square).